qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/SnapshotReplicationTest.java

```diff
@@ -66,6 +68,7 @@
   @Before
   public void init() {
     client = clientRule.getClient();
+    clusteringRule.getClock().pinCurrentTime();
```

Comment:
We should not pin the time in integration tests. This can lead to weird problems :sweat_smile: